### ISSUE TITLE
Use kops

### ISSUE THREAD

**Issue by drewwells** [cablespaghetti/kubeadm-aws#11]
Have you considered swapping out for kops? It shouldn't be difficult to replicate this setup in a kops cluster.

**Comment by silashansen** [comment]
Cool repo, but I was thinking the same thing...

**Comment by cablespaghetti** [comment]
Yeah this is a bit of a "because I can" project. If I remember right, at the time when I started this I don't think you could use spot instances for a master node on kops, maybe you can now.

**Comment by drewwells** [comment]
they appear to support this option now https://github.com/kubernetes/kops/blob/master/docs/instance_groups.md#creating-a-instance-group-of-mixed-instances-types-aws-only

Does pose an interesting question... how do you handle losing the one master's etcd? Is this okay or is there a backup ?
